tests/Tests/500__jupyterhub/test.robot

```diff
@@ -107,3 +98,13 @@ Start New Pytorch Build
     ${new_buildname}=  Start New Build    namespace=redhat-ods-applications    buildconfig=s2i-pytorch-gpu-cuda-11.4.2-notebook
     Wait Until Build Status Is    namespace=redhat-ods-applications    build_name=${new_buildname}   expected_status=Running
     [Return]    ${new_buildname}
+
+Verify Modal Does Not Get Stuck
```

Comment:
Could this keyword be renamed to be more descriptive, like for example using the polarion test tittle? `Verify Notebook Spawner Modal Does Not Get Stuck When Requesting Too Many Resources To Spawn Server`

Reply:
Done